docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -172,6 +172,21 @@ The meter plugin provides the advanced metrics collections, which are not a part
   * [Alibaba Druid](https://github.com/alibaba/druid) 1.x
   * [HikariCP](https://github.com/brettwooldridge/HikariCP) 3.x -> 4.x
   * [C3P0](https://github.com/swaldman/c3p0) 0.9.0 -> 0.10.0
+* Database
```

Comment:
I think this should be at lower level. This is a part connection pool.

Reply:
OK.